### PR TITLE
Remove migration and mongodb logic

### PR DESCRIPTION
**Pull request by regiskuckaertz**

### CODE REVIEW

File: riff-raff/app/AppComponents.scala [hunk 1]
```diff
@@ -50,15 +49,7 @@ class AppComponents(context: Context) extends BuiltInComponentsFromContext(conte
...
-  private lazy val mongoStore: DataStore = {
-    val dataStore = new MongoDatastoreOps(config).buildDatastore().getOrElse(new NoOpDataStore(config))
-    log.info(s"Persistence datastore initialised as $dataStore")
-    dataStore
-  }
-
-  private lazy val postgresStore: DataStore = {
+  lazy val datastore: DataStore = {
     new PostgresDatastoreOps(config).buildDatastore()
   }
```
Comment (marialivia16) [inline riff-raff/app/AppComponents.scala]: No need to have the `{}` anymore.

Comment (marialivia16) [inline riff-raff/app/AppComponents.scala]: No need to have the `{}` anymore

File: riff-raff/app/persistence/postgresdb.scala [hunk 2]
```diff
@@ -12,16 +12,36 @@ import play.api.libs.json._
...
+
   // Table: auth(email: String, content: jsonb)
   def getAuthorisation(email: String): Either[Throwable, Option[AuthorisationRecord]] = logExceptions(Some(s"Requesting authorisation object for $email")) {
     DB readOnly { implicit session =>
       sql"SELECT content FROM auth WHERE email = $email".map(AuthorisationRecord(_)).single.apply()
     }
   }
 
-  def getAuthorisationList(pagination: Option[deployment.PaginationView]): Either[Throwable, List[AuthorisationRecord]] = logExceptions(Some("Requesting list of authorisation objects")) {
+  def getAuthorisationList: Either[Throwable, List[AuthorisationRecord]] = logExceptions(Some("Requesting list of authorisation objects")) {
```
Comment (marialivia16) [inline riff-raff/app/persistence/postgresdb.scala]: Is pagination used in the old mongo API? If it is, then we should keep it and replicate behaviour.

Comment (regiskuckaertz) [reply riff-raff/app/persistence/postgresdb.scala]: It wasn't used, I added it for having a consistent migration script.

**Comment by regiskuckaertz** [comment]
Yes indeed 🤗  I'm just not sure how to display the `CollectionStats`. Do you know?

**Comment by marialivia16** [comment]
Not sure, are the metrics sent to CloudWatch?

**Comment by regiskuckaertz** [comment]
I don't think so 🤔 It's supposed to be visible locally if [I understand correctly](https://github.com/guardian/guardian-management). Will keep looking.

**Comment by regiskuckaertz** [comment]
Ha, finally works:
```
{
    "group" : "postgres",
    "name" : "data_size",
    "type" : "gauge",
    "title" : "PostgreSQL data size",
    "description" : "The size of the data held in postgres tables",
    "value" : "1052827648"
  }, {
    "group" : "postgres",
    "name" : "storage_size",
    "type" : "gauge",
    "title" : "PostgreSQL storage size",
    "description" : "The size of the storage used by the PostgreSQL tables",
    "value" : "1727881216"
  }, {
    "group" : "postgres",
    "name" : "deploys_collection_count",
    "type" : "gauge",
    "title" : "Deploys collection count",
    "description" : "The number of rows in the deploys table",
    "value" : "417232"
  }
```

**Pull request merged**